lib/vagrant-vbguest/installers/linux.rb

```diff
@@ -166,21 +175,21 @@ def systemd_tool
         end
       end
 
-      # Checks for the correct location of the 'vboxadd' tool.
+      # Checks for the correct location of the tool provided.
       # It checks for a given list of possible locations. This list got
       # extracted from the 'VBoxLinuxAdditions.run' script.
       #
-      # @return [String|nil] Absolute path to the +vboxadd+ tool,
+      # @return [String|nil] Absolute path to the tool,
       #                      or +nil+ if none found.
-      def vboxadd_tool
+      def find_tool(tool)
```

Comment:
yup apologies that's my fault. I've made that edit locally but for some reason it didn't make it into the commit. I'll update that.